crates/dapf/src/cli_parsers.rs

```diff
@@ -0,0 +1,324 @@
+//! Human friendly parsers for common types of parameters to DAP functions.
+// Copyright (c) 2024 Cloudflare, Inc. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
The copyright notice should be at the top of the file. The module docstring should have newlines around it.